app/src/main/java/org/schabi/newpipe/util/ExtractorHelper.java

```diff
@@ -204,6 +217,33 @@ public static Single<InfoItemsPage<StreamInfoItem>> getMoreKioskItems(final int
                 KioskInfo.getMoreItems(NewPipe.getService(serviceId), url, nextPage));
     }
```

Comment:
Shouldn’t this have fallback code as well? Might fail when scrolling down otherwise